packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts

```diff
@@ -62,22 +63,57 @@ export abstract class CosmosSdkBaseAdapter<T extends CosmosChainTypes> implement
   async getAccount(pubkey: string): Promise<chainAdapters.Account<T>> {
     try {
       const caip = this.getCaip2()
-      const { chain, network } = caip2.fromCAIP2(caip)
       const { data } = await this.providers.http.getAccount({ pubkey })
 
+      const delegations = data.delegations.map<chainAdapters.cosmos.Delegation>((v) => ({
+        assetId: this.assetId,
+        amount: v.balance.amount,
+        validator: {
+          address: v.validator
+        }
+      }))
```

Comment:
I would prefer these transform functions be moved into separate functions so make them more testable and reusable.

```ts
const transformDelegations = (delegations: chainAdapters.cosmos.Delegation) => ({
assetId: this.assetId,
        amount: v.balance.amount,
        validator: {
          address: v.validator
        }
      }))

const delegations = data.delegations.map<chainAdapters.cosmos.Delegation>(transformDelegations)
```

Reply:
Not sure it provides a ton of value, but I can. It is already reusable in the sense it exists in the base adapter, and testing `getAccount` seems sufficient to me once we actually add tests.